### ISSUE TITLE
using local push with app closed???

### ISSUE THREAD

**Issue by eros2187** [katzer/cordova-plugin-local-notifications#750]
Hi I wanted to know if it was possible to use push local Lapp closed ?

I A Function That reminds me EVERY 15 Minutes A Function
narrative function but if the App is closed in background not just shut it does not work and can come whatsapp rate Something ?? Thank you

**Comment by slightlydiff** [comment]
Does anyone know the answer to this - basically when an app is backgrounded in IOS will the trigger events allow some code in teh app to be executed?

The README.MD implies this is correct and Apple documents a mechanism for allowing this with content=1 in the local push notification but this plugin doesn't seem to work like this with IOS9.

Any helps would be great please

**Comment by Tawpie** [comment]
@slightlydiff At the moment the notification fires, your code is NOT executed but something in the plugin may get called. The user has to manually 'wake your app up' by touching the notification in the notification pulldown.

**Comment by katzer** [comment]
__local__ and __push__-notifications are two different topics. Pls don't merge them. What @Tawpie mentioned is right. On iOS the app cannot wakeup itself. The app comes into the play when a user has pressed the button. Everything before is done by OS.

**Comment by Heshyo** [comment]
@katzer I've seen some app on iOS and Android that work like an alarm clock: you configure when you want it to wake you up, and the app will automatically start playing some radio. Do you know if it's done through a completely different mechanism as your plugin?

**Comment by Heshyo** [comment]
@katzer Actually I now found https://github.com/katzer/cordova-plugin-local-notifications/issues/899 that wishes to do the same, ie start the app without the user having to click the notification.

I also found another plugin that's supposed to work like that https://github.com/wnyc/cordova-plugin-wakeuptimer . The problem is that from the comments that plugin has several issues. But I also saw @souly1 mentioning your plugin and suggesting a way to modify it to make it work that way, at least on Android https://github.com/wnyc/cordova-plugin-wakeuptimer/issues/13#issuecomment-195037425 So maybe it's possible to add that functionality (at least on Android) without much modifications?
